.github/workflows/ci.yml

```
@@ -200,12 +200,12 @@ jobs:
           key: phpstan-${{ github.ref_name }}
 
       - name: Clear previous version of PHPStan main cache
-        if: github.ref == 'refs/heads/$({ env.MAIN_BRANCH }}'
+        if: ${{ github.ref == format('refs/heads/{0}', env.MAIN_BRANCH) }}
```

```suggestion
        if: ${{ github.ref_name == github.event.repository.default_branch }}
```
What do you think about it?

---

That would be great, didn't know this was possible!

---

Done.